qa/integration/integration_tests.gemspec

```diff
@@ -18,8 +18,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'manticore'
   s.add_development_dependency 'stud'
   s.add_development_dependency 'pry'
-  s.add_development_dependency 'logstash-devutils'
```

Comment:
was there a reason this was removed ?  

I had to add it back to get it to work.

Reply:
Really? with the changes I made it should have worked.

We really shouldn't depend on this for the integration spec suite.

I'll run the suite again and see if we can patch the tests.